content/quests/events/demevent_bog_vaccine.lua

```diff
@@ -17,129 +17,169 @@ QDEF:AddConvo()
+                    !hesh_greeting
+                    Ah, as the Waterlogged Tomes decreed, it was to be kept secret from the public, as it could eventually cause the heretics to destroy it.
                 player:
-                    Well...
+                    Heretics wouldn't have any reason to destroy it, though.
                 agent:
-                    There you have it.
-                    That is the answer to where this vaccine comes from.
+                    !angrypoint
+                    Someone who questions the Waterlogged Tomes! Have you no shame?
+                player:
+                    My shame's perfectly intact, but fine. Don't tell me how it works, then.
             ]],
             DIALOG_CONVINCE_FAILURE = [[
-                player:
-                    [p] Look, I'm just asking the questions.
                 agent:
-                    Do you doubt Hesh's ability to produce a countermeasure to its threat quickly?
+                    Well, it's a mixture of Salt Water, Lumin, some grains of sand and a whole heaping lot of "none of your business".
                 player:
-                    Well, I-
+                    !crossed
+                    I believe this is my business if you're going to inject something into my veins.
                 agent:
-                    I suggest you ask no further if you don't wish to insult Hesh itself.
+                    And I believe in Hesh, and it's wrath will come down on you if you keep questioning this.
+                player:
+                    Ah, fine. Forget I asked.
             ]],
             OPT_REFUSE = "Refuse the vaccine",
             DIALOG_REFUSE = [[
                 {asked_info?
                     player:
-                        [p] You call yourself a priest, yet you use the tools of the spark.
-                        Have you no honor? No faith in the Cult?
+                        !hips
+                        I don't think i'll be getting shot up with this vaccine you called "not consumer-friendly".
+                    agent:
+                        !angry_point
+                        Do you wish for eternal damnation?
+                    player:
+                        Tough words for the Priest using deemed heretical tools to do holy work.
                     agent:
-                        I- I assure you, my faith perfectly fine.
+                        It's out of necces-
                     player:
-                        Then you should know not to inject the heretic's product into someone's body.
+                        Neccessity to keep the voter base alive. Neccessity to keep <i>your<\> voter base alive.
                     agent:
-                        Alright then. Have it your way.
-                        Just be careful not to get infected by the bog.
+                        !point
+                        I-
+                        Fine! Let that damned bog drag your soul down with it!
```

Comment:
The idea is that when you get the intel, the priest isn't mad at you because the source of this vaccine is genuinely questionable (and the priest starts to doubting their actions), but they do warn you to watch out for the bog if you refuse.

Reply:
Toned back the priest's language, so they're a little more remorseful if you refuse the vaccine